common/build.gradle.kts

```diff
@@ -66,9 +67,11 @@ kotlin {
         }
 
         androidMain.dependencies {
-            implementation(libs.ktor.client.android)
+            implementation(libs.ktor.client.okhttp)
```

Comment:
better than Android HttpClient. Http/2.

Not used on JVM, since HttpClient is modern.